src/coreComponents/common/unitTests/testCaliperSmoke.cpp

```diff
@@ -0,0 +1,58 @@
+/*
+ * ------------------------------------------------------------------------------------------------------------
+ * SPDX-License-Identifier: LGPL-2.1-only
+ *
+ * Copyright (c) 2018-2022 Lawrence Livermore National Security LLC
+ * Copyright (c) 2018-2022 The Board of Trustees of the Leland Stanford Junior University
+ * Copyright (c) 2018-2022 TotalEnergies
+ * Copyright (c) 2019-     GEOSX Contributors
+ * All rights reserved
+ *
+ * See top level LICENSE, COPYRIGHT, CONTRIBUTORS, NOTICE, and ACKNOWLEDGEMENTS files for details.
+ * ------------------------------------------------------------------------------------------------------------
+ */
+
+#include <gtest/gtest.h>
+
+#include <cstdlib>
+#include <string>
+
+#include "common/TimingMacros.hpp"
+#include <adiak.hpp>
+
+TEST( CaliperSmoke, SmokeTest )
```

Comment:
Maybe you could have used the `SetUp` / `TearDown` feature here?
It looks like that's the kind of pattern you're trying to achieve around the `env`.